demisto_sdk/commands/test_content/IntegrationsLock.py

```diff
@@ -240,7 +243,7 @@ def unlock_integrations(
         test_playbook (TestPlaybook): The test playbook instance we want to test under the lock's context
         storage_client: The GCP storage client
     """
-    locked_integrations = [integration.name for integration in integrations_to_unlock]
+    locked_integrations = [getattr(integration, 'name', integration) for integration in integrations_to_unlock]
```

Comment:
why?

Reply:
because in some cases the **integrations_to_unlock** contains strings that appends in **create_lock_files** method:
<img width="501" alt="image" src="https://github.com/demisto/demisto-sdk/assets/46249224/310d5269-a239-4b4d-8d44-db2c38bcb585">